examples/up.conf.example

```diff
@@ -65,6 +65,28 @@
   interval   = "2m"
   timeout    = "10s"
 
+[datadog]
+  # How often to poll UniFi and report to Datadog.
+  interval = "2m"
+
+  # To disable this output plugin
+  disable = true
```

Comment:
maybe we should mark the other outputs default `disable = true` as well? I currently see `prometheus` && `influx` default enabled. (not sure about `loki`)

Reply:
What is the behavior of this module if this is left enabled but not used? For prometheus, it just opens an http listener that never does anything else, so it's "ok" to leave it enabled. For influxdb, it's the original storage method and the default has always been enabled (at one point it could not be disabled). It's also the most common storage backend among the users. Folks not using it have to explicitly disable it. I don't want there to be two output plugins that must be disabled when not in use. If this module "does nothing" when enabled but not used, then defaulting to enabled is OK.

Reply:
good question - it would initiate a statsd reporter, but the client code itself is non-blocking and would just fail to report if the agent wasn't available to receive the statsd data.